f2/src/components/text-area/__tests__/TextArea.test.js

```diff
@@ -2,23 +2,10 @@ import React from 'react'
 import { ThemeProvider } from 'emotion-theming'
 import { TextArea } from '../'
 import { render, cleanup } from '@testing-library/react'
-import { mount } from 'enzyme'
 
 describe('<TextArea />', () => {
   afterEach(cleanup)
 
-  it('properly renders as a textarea tag', () => {
-    let wrapper = mount(
-      <form>
-        <label>
-          Test
-          <TextArea />
-        </label>
-      </form>,
-    ).find('textarea')
-    expect(wrapper.is('textarea')).toBeTruthy()
-  })
-
```

Comment:
I think these two tests will now pass with the other changes you’ve made - can you add them back in and see?